app/exec/extension/_lib/vsix-manifest-builder.ts

```diff
@@ -794,22 +794,29 @@ export class VsixManifestBuilder extends ManifestBuilder {
 			});
 		}
 		return contentTypePromise.then(() => {
+			let seenPartNames = new Set();
 			Object.keys(this.files).forEach(filePath => {
 				if (this.files[filePath].contentType) {
+					let partName = "/" + toZipItemName(this.files[filePath].partName);
 					contentTypes.Types.Override.push({
```

Comment:
Should we also check seenPartNames for partName here?

Reply:
we could, and I am fine doing that, but the specific issue that was causing the issue was that the keys in this.files was absolute paths, but the paths in the _additionalPackagePaths were relative paths and so that is why they were able to get into the map twice.

Reply:
Cool. I am fine either way.